### PR TITLE
TC_02.04.024 | Dashboard > Verify links on left command panel

### PR DESCRIPTION
**Pull request by IliaKhudiakov**
https://trello.com/c/QI8SKBjT/317-tc0204024-dashboard-verify-links-on-left-command-panel

### CODE REVIEW

File: cypress/fixtures/leftSideItems.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "Item" : ["New Item", "People", "Build History", "Manage Jenkins", "My Views"]
+}
```
Comment (S-Lanchik) [inline cypress/fixtures/leftSideItems.json:3]: Why do you put the same data to two different json files? Delete this file, and use this data from another file. Moreover, the same data is already in the main branch. Use files that are already created by other people.

**Comment by S-Lanchik** [comment]
Don't forget about rule of the project: one PR - one test

File: cypress/fixtures/leftSideItems.json (hunk 1 above)
Comment (IliaKhudiakov) [reply cypress/fixtures/leftSideItems.json:3]: Just delete fixtures file.

Comment (IliaKhudiakov) [reply cypress/fixtures/leftSideItems.json]: Do I need to split this test to 5 different?

File: cypress/e2e/VerifyWelcomedMessageAndGuideOnHomepage.cy.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import quideMessage from "../fixtures/welcomeAndGuideMessage.json"
+
+describe ('TC_02.03.015 | Verify welcomed message', () => {
+    it('Verify welcomed message and quick guide on homepage', function() {
+        cy.get('h1').should('be.visible')
+        cy.get('h1').should('have.text', welcomeMessage.greeting)
+        cy.get('p').should('be.visible')
+        cy.get('p').should('have.text', quideMessage.quickquide)
+    })
+})
```
Comment (S-Lanchik) [inline cypress/e2e/VerifyWelcomedMessageAndGuideOnHomepage.cy.js:13]: This test is in your PR 189, delete it from this PR.

File: cypress/fixtures/leftSideItems.json (hunk 1 above)
Comment (S-Lanchik) [reply cypress/fixtures/leftSideItems.json]: You could, but it is not necessary. Just follow the rule: one PR - one test. You have three test in this PR.

**Review by Rom-T — APPROVED** [review]

**Review by S-Lanchik — APPROVED** [review]

**Pull request merged**